addusers.py

```diff
@@ -467,6 +496,17 @@ def parse_rows(rows, select_user=None):
                    "You may need to close multiple tickets.").format(
                        len(copy_index), args.user)
         copy_rows = [r for r in rows if rows.index(r) in copy_index]
+        max_length = len(header)
+        user_header_column = header.index(email_column)
+        new_project_header_column = header.index(project_name_column)
+        existing_project_header_column = header.index(join_project_column)
+        for row in copy_rows:
+            if len(row) != max_length:
```

Comment:
You don't need the expansion when `if row[user_header_column] in user_dict`is false, so do that check first to avoid unnecessary work.